web_poet/utils.py

```diff
@@ -12,6 +12,15 @@
 from url_matcher import Patterns
 
 
+def callable_has_argument(obj, kwarg):
```

Comment:
Found this: https://docs.python.org/3/faq/programming.html#faq-argument-vs-parameter :) Should we call it "callable_has_parameter", and rename "kwarg" to something else like "name"?